### PR TITLE
chore(testing): use ngx-tools createComponent utility

### PR DESCRIPTION
**Pull request by atlwendy**
Removed all the local defined `createComponent` methods and use ngx-tools utility instead.

### CODE REVIEW

File: demo/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,25 +1,31 @@
-import { async, TestBed } from '@angular/core/testing';
+import { async, TestBed, ComponentFixture } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 
 import { AppComponent } from './app.component';
+import { Component } from '@angular/core';
```
Comment (shani-terminus) [inline demo/app/app.component.spec.ts]: Imports should be alphabetized. You can run `yarn run lint:app:ts`.

File: terminus-ui/date-range/src/date-range.component.spec.ts [hunk 2]
```diff
@@ -239,17 +239,11 @@ describe(`TsDateRangeComponent`, function() {
 
 // TODO: Move to ngx-tools (and all other instances of this utility)
```
Comment (shani-terminus) [inline terminus-ui/date-range/src/date-range.component.spec.ts]: update or remove comment

**Review by shani-terminus — CHANGES_REQUESTED** [review]
Looks good, just a couple things to clean up

File: demo/app/app.component.spec.ts (hunk 1 above)
Comment (benjamincharity) [reply demo/app/app.component.spec.ts]: This should also be up with 3rd party imports

File: terminus-ui/table/src/table.component.spec.ts [hunk 3]
```diff
@@ -517,15 +525,5 @@ export function createComponent<T>(component: Type<T>, providers: Provider[] = [
...
-      {
-        provide: TsWindowService,
-        useClass: TsWindowServiceMock,
-      },
-      ...providers,
-    ],
-  }).compileComponents();
-
-  return TestBed.createComponent<T>(component);
+    );
```
Comment (benjamincharity) [inline terminus-ui/table/src/table.component.spec.ts]: This should be de-indented

**Review by benjamincharity — CHANGES_REQUESTED** [review]

**Review by benjamincharity — APPROVED** [review]

**Pull request merged**